### PR TITLE
Add support for `estimated_cell_count` in `project.json` (#53)

### PR DESCRIPTION
**Pull request by dsotirho-ucsc**
https://github.com/databiosphere/hca-metadata-api/issues/53

### CODE REVIEW

**Review by amarjandu — APPROVED** [review]

File: test/test.py [hunk 1]
```diff
@@ -520,7 +520,7 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files,
         return bundle
 
     def test_canned_staging_area(self):
-        ref = 'de355cad77ea7988040b6f1f5f2eafae58f686a8'
+        ref = '55628953e4b3a24a7d7798569b6082032bd07a6b'
```
Comment (hannes-ucsc) [inline test/test.py:523]: Assert value of added attribute.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
PR title deviates from commit title for no reason.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
Also no reason to split the commits.

**Review by hannes-ucsc — APPROVED** [review]

**Comment by hannes-ucsc** [comment]
@jessebrennan, as operator you *should* be able to cut a release for this. We may also hold off with the release for a couple days to wait for @amarjandu's fix for #54 and release both at the same time.

**Pull request merged**